src/common/config.ts

```diff
@@ -1,14 +1,28 @@
 import * as uuidv4 from 'uuid/v4';
 import { loadConfig } from 'snyk-config';
+import * as fs from 'fs';
 
 const config: Record<string, any> = loadConfig(__dirname + '/../..', {
   secretConfig: process.env.CONFIG_SECRET_FILE,
 });
 
+const namespacesFilePath = '/etc/config/namespaces';
+
+function loadExcludedNamespaces(): string[] | null {
+  try {
+    const data = fs.readFileSync(namespacesFilePath, 'UTF-8');
+    const namespaces: string[] = data.split(/\r?\n/);
```

Comment:
```suggestion
    const namespaces: string[] = data.split(/\r?\n/).filter(Boolean);
```
It will filter out any empty lines if there will be case like that